### DISCUSSION TITLE
crash

### DISCUSSION THREAD

**Discussion by ouen-cloud** [JordanSantiagoYT/FNF-JS-Engine#410] [General]
crash
![crash](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/174063540/56042612-e9cc-48d4-a1df-2a0a1e738a49)

**Comment by moxie-coder** [comment]
update to the latest version

**Comment by moxie-coder** [comment]
also wrong category for this discussion + this is an issue, not an discussion
